HW3.Rmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "HW3"
+author: "Sherzod Tokamov"
+date: "7/20/2020"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(ggplot2)
+angles <- read_csv("angles.csv", stringsAsFactors = FALSE) #read my csv file
```

Comment:
I think read.csv() might work better here; when I tried running your code with read_csv(), it required the readr package and tidyverse library, and told me the stringsAsFactors argument was not used.